gradle/mwe2-workflows.gradle

```diff
@@ -12,7 +12,7 @@ dependencies {
 	if (!name.endsWith('bootstrap')) {
 		mwe2Compile project(':org.eclipse.xtext.xtext.generator')
 	}
-	mwe2Runtime "org.eclipse.emf:org.eclipse.emf.mwe2.launch:$versions.emfMwe2"
+	mwe2Runtime "org.eclipse.emf:org.eclipse.emf.mwe2.launch"
```

Comment:
When now variable resolution is involved single quotes are preferred

Reply:
Not important